test/lib/adapters/FileSystem_read.js

```diff
@@ -94,7 +94,7 @@ test("glob resources from application.a with directory exclude", async (t) => {
 	});
 
 	await readerWriter.byGlob("/!(pony,unicorn)/**").then(function(resources) {
-		t.deepEqual(resources.length, 2, "Found exactly two resource");
+		t.deepEqual(resources.length, 3, "Found exactly three resource");
```

Comment:
test/fixtures/application.a/webapp/empty.js can be removed again and this change reverted

Reply:
done